wandb/sdk/internal/sender.py

```diff
@@ -243,6 +243,8 @@ def send_request_stop_status(self, record):
     def debounce(self) -> None:
         if self._config_needs_debounce:
             self._debounce_config()
+        if self._fs:
+            self._fs.process_queue()
```

Comment:
Since we have no thread we're counting on another thread calling debounce.  How has the frequency at which we're processing events from the queue changed?